README.md

```diff
@@ -204,15 +206,15 @@ from 0-5 (~2.6 bits of information)
 
 ### Register a conversion algorithm
 
-When the user agent receives a conversion registration on a URL matching
-the conversiondestination eTLD+1, it looks up all impressions in storage that
-match <reportingorigin, conversiondestination>.
+When the browser receives a conversion registration on a URL matching
+the `conversiondestination` eTLD+1, it looks up all impressions in storage that
+match <`reportingorigin`, `conversiondestination`>.
 
 The most recent matching impression is given an `attribution-credit` of value 100. All other matching impressions are given an `attribution-credit` of value of 0.
 
 For each matching impression, schedule a report. To schedule a report,
 the browser will store the 
- {reporting origin, conversiondestination domain, impression data, [decoded](#metadata-encoding) conversion-metadata, attribution-credit} for the impression.
+ {reporting origin, `conversiondestination` domain, impression data, [decoded](#metadata-encoding) conversion-metadata, attribution-credit} for the impression.
```

Comment:
Should we use `impression-data` as it is also an attribute?

Reply:
Yes, fixed - also edited `reportingorigin`.